test/k8sT/Nightly.go

```diff
@@ -226,6 +227,90 @@ var _ = Describe("NightlyEpsMeasurement", func() {
 			}
 		}, 1)
 	})
+
+	Context("Test long live connections", func() {
+		getServer := func(port string) string {
+			return fmt.Sprintf("nc -p %s -lk -v", port)
+		}
+
+		getClient := func(ip, port, filePipe string) string {
+			return fmt.Sprintf(
+				"rm %[1]s; touch %[1]s; tail -f %[1]s 2>&1 | nc -v %[2]s %[3]s",
```

Comment:
FYI there are multiple implementations of `nc` that are used in different distros and they may/may not have the same behaviour with different arguments. In OVS we ended up parameterizing the arguments to pass into `nc` to ensure that it would exit with a five second timeout, and the arguments had to be defined differently on different distros. I'd suggest testing to making sure that if this fails for some reason, that it eventually exits.

https://github.com/openvswitch/ovs/blob/b1db9dd6747cf8962ebcca4fc15dd084728b0f79/tests/atlocal.in#L159-L166